src/classes/ADDR_HouseholdAcc_TEST.cls

```diff
@@ -1698,6 +1698,7 @@ public with sharing class ADDR_HouseholdAcc_TEST {
         // insert new contacts to the HH without any addresses
         List<Contact> listCon = UTIL_UnitTestData_TEST.getMultipleTestContacts(2);
         for (Integer i = 0; i < 2; i++) {
+            listCon[i].LastName = 'ToAvoidDupeRule' + i;
```

Comment:
Changes in this class are due to Duplicate Rules errors. Don't really have anything to do with TDTM...